app/util/variables.ts

```diff
@@ -22,6 +22,8 @@ export interface HarmonyRelatedUrl {
 
 interface VariableInfo {
```

Comment:
I'm wondering if the name of this interface is confusing since it has more info than just what is used for variables.